packages/crayons-extended/custom-objects/src/components/platform-table/platform-table.scss

```diff
@@ -41,6 +41,8 @@
 }
 
 .table {
+  height: var(--fw-platform-table-width, 100%);
```

Comment:
pls add css variables at the top so that it gets picked up in the docs. like
```/**
  @prop ----fw-platform-table-width: description
*/

Reply:
Thanks for the review @arvindanta , have updated the PR